runtime/constants.go

```diff
@@ -46,6 +46,11 @@ const (
 	clientSystemErrors = "client.system-errors"
 	clientLatency      = "client.latency"
 	clientLatencyHist  = "client.latency-hist"
+
+	// clientHTTPUnmarshalError is the metric for tracking errors due to unmarshalling json responses
+	clientHTTPUnmarshalError = "client.http-unmarshal-error"
+	// clientTchannelReadError is the metric for tracking errors in reading tchannel response
+	clientTchannelReadError = "client.tchannel-read-error"
```

Comment:
technically `read` may be more appropriate but when comparing metrics, it helps if they are consistent. so I would call this `tchannel-unmarshal-error` along with the var name too.